src/components/Menu/Menu.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'clsx';
+import styles from './Menu.scss';
+
+export const Menu = ({children, isDisplay, maxWidth, position, className}) => {
+    if (isDisplay) {
+        return (
+            <menu style={{maxWidth: maxWidth, ...position}} className={classnames(styles.menu, className)}>
+                {children}
+            </menu>
+        );
+    }
+
+    return null;
+};
+
+Menu.defaultProps = {
+    maxWidth: 'fit-content',
+    position: {top: '0', left: '0'}
+};
+
+Menu.propTypes = {
+    /**
```

Comment:
We will need `onMouseEnter()` , `onMouseLeave()` , and ideally `onClosed()` if there's a fade out, to be called after the transition